src/components/Tables/DataTable/fragments/inputs/TableInputTag.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <Tag v-if="!edit" :value="data" />
+  <InputText v-else v-model="data" />
+</template>
+
+<script setup lang="ts">
+import Tag from "primevue/tag";
+import InputText from "primevue/inputnumber";
+
+export type TableInputNumberProps = { edit: boolean };
+defineProps<TableInputNumberProps>();
+
+const data = <any>defineModel();
```

Comment:
me parece que esse `data` será sempre uma `string`. É isso mesmo? Acho que podemos tipá-lo corretamente também.
(E não se esqueça da tipagem explícita)

```suggestion
const data = defineModel<string>(); // verificar se é só string mesmo
```